taxonomy-editor-frontend/src/pages/editentry/ListEntryChildren.jsx

```diff
@@ -13,13 +13,13 @@ import ISO6391 from 'iso-639-1';
 import { ENTER_KEYCODE } from "../../constants";
 
 
-const ListEntryChildren = ({url, setUpdateNodeChildren}) => {
+const ListEntryChildren = ({url, urlPrefix, setUpdateNodeChildren}) => {
     const [relations, setRelations] = useState(null);
     const [newChild, setNewChild] = useState(null);
     const [newLanguageCode, setNewLanguageCode] = useState(null);
-    const [open, setOpen] = useState(false); // Used for Dialog component
-    const [btnDisabled, setBtnDisabled] = useState(true) // For enabling or disabling Dialog button
-    const [isValidLC, setisValidLC] = useState(false); // Used for validating a new LC
+    const [openDialog, setOpenDialog] = useState(false); // Used for Dialog component
+    const [isValidLanguageCode, setIsValidLanguageCode] = useState(false); // Used for validating a new LC
+    const greyHexCode = "#808080";
```

Comment:
this should live in the `constants` file, not in here as this color is used several times. So you define it once and use it every time you need it. Defining it several times makes it difficult to ensure consistency

Reply:
Yep, changed it